src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/CompareType.java

```diff
@@ -88,7 +94,7 @@ public static CompareType findByOperator(char operator) {
         return PLAIN;
     }
 
-    private CompareUtil util;
+    CompareUtil util;
```

Comment:
with the existing setup it was necessary for the test. With your suggestion it won't be any more. Correct.